android/modules/network/src/java/ti/modules/titanium/network/NetworkModule.java

```diff
@@ -479,6 +484,24 @@ public void removeAllHTTPCookies()
 		cookieStore.removeAll();
 	}
 
+	/**
+	 * Returns true if push notifications are allowed in the app settings
+	 * @return boolean if push notifications are allowed in the app settings
+	 */
+	@RequiresApi(api = Build.VERSION_CODES.S)
+	@Kroll.getProperty
+	public boolean remoteNotificationsEnabled()
+	{
+		if (Build.VERSION.SDK_INT < Build.VERSION_CODES.S) {
+			return true;
+		}
+
+		Context context = TiApplication.getInstance();
+		int result = context.checkSelfPermission(Manifest.permission.POST_NOTIFICATIONS);
+		result &= context.checkSelfPermission(Manifest.permission.POST_NOTIFICATIONS);
```

Comment:
Silly question, but why check the same `Manifest.permission.POST_NOTIFICATIONS` value twice?

Reply:
🤦 you are right. Was testing stuff before, will remove that!

Reply:
Happens!